index.js

```diff
@@ -0,0 +1,92 @@
+const swithBtn = document.getElementById("twelveHourBtn");
+
+let twelveHourBtn = document.getElementById("twelveHourBtn");
+let getTime = document.getElementById("time");
+let isTwelveHour = true;
+let amPm = " AM";
+
+function checkTime(i) {
+  if(i < 10) {
+    i = "0" + i;
+  }
+  return i;

```

Comment:
```suggestion
  return i < 10 ? `0${i}` : i;

```


also use arrow func with implicit return